src/lib/swapper/swappers/ThorchainSwapper/utils/addSlippageToMemo.ts

```diff
@@ -12,11 +12,13 @@ import { assertIsValidMemo } from './makeSwapMemo/assertIsValidMemo'
 
 export const addSlippageToMemo = (
   expectedAmountOutThorBaseUnit: string,
-  quotedMemo: string,
+  quotedMemo = '',
   slippageBps: BigNumber.Value,
   isStreaming: boolean,
   chainId: ChainId,
 ) => {
+  if (!quotedMemo) throw new Error('no memo provided')
```

Comment:
See suggestion above. Currently, if the `memo` arg is `undefined`, it'll default to `''`, which is falsy, so this will never actually throw.

Reply:
empty string would throw, but explicit undefined is better. i defaulted to using an optional and didn't want to change arg order, but `string | undefined` best

Reply:
Ugh you're right, sorry - my early morning brain was broken.